pkg/memio/memio_ports.go

```diff
@@ -12,45 +12,52 @@ import (
 	"github.com/u-root/u-root/pkg/ubinary"
 )
 
-type MemIOReader interface {
+// Reader is the interface for reading from memory and IO ports.
+type Reader interface {
 	Read(UintN, int64) error
 }
 
-type MemIOWriter interface {
+// Writer is the interface for writing to memory and IO ports.
+type Writer interface {
 	Write(UintN, int64) error
 }
 
-type MemIOReadWriteCloser interface {
-	MemIOReader
-	MemIOWriter
+// ReadWriteCloser implements io.ReadWriteCloser
+type ReadWriteCloser interface {
+	Reader
+	Writer
 	io.Closer
 }
 
-type MemIOPort struct {
+// Port implements memory and IO port access via an os.File.
+type Port struct {
 	*os.File
 }
 
-func (m *MemIOPort) Read(out UintN, addr int64) error {
+var _ ReadWriteCloser = &Port{}
+
+func (m *Port) Read(out UintN, addr int64) error {
```

Comment:
Maybe add a comment above this function as well, now that you're adding them to other functions in this file.